prac_06/guitars.py

```diff
@@ -0,0 +1,26 @@
+"""
+CP1404/CP5632 Practical
+Guitars
+
+Expected Time: 40 Minutes
+Actual Time:  20 Minutes
```

Comment:
Nice! You are more efficient than anticipated.

Reply:
Thanks :)